src/theme/login/webauthn-authenticate.ftl

```diff
@@ -0,0 +1,118 @@
+<#import "template.ftl" as layout>
```

Comment:
Not super sure where this comment belongs but I think it would be helpful to update the theming/buttons around the webauthn during registration. Currently it's not super clear what this register button is for (and I'm not sure if the sign out button should be there too?).

![](https://files.slack.com/files-pri/T088J8BL3PD-F08KDGH0S3C/screenshot_2025-03-25_at_3.35.03___pm.png)

Reply:
LGTM!